databaseQueries/programUsers.js

```diff
@@ -0,0 +1,35 @@
+module.exports = class programUsers {
```

Comment:
@ankitpws - Please add basic information about the file here.

Reply:
Added basic info